MekHQ/src/mekhq/campaign/personnel/Person.java

```diff
@@ -3631,4 +3652,19 @@ public boolean hasChildren() {
         
         return hasKids;
     }
+    
+    // Returns the ransom value of this individual
+    // Useful for prisoner who you want to ransom or hand off to your employer in an AtB context
```

Comment:
JavaDoc comments should be preceeded with `/**` and end with `*/`
http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#format